docs/code/MlNetCookBook.md

```diff
@@ -146,8 +147,8 @@ var data = mlContext.Data.LoadFromTextFile<InspectedRow>(dataPath,
 You can again use the `TextLoader`, and specify an array of files to its Load method.
 The files need to have the same schema (same number and type of columns) 
 
-[Example file1](../../test/data/adult.train):
-[Example file2](../../test/data/adult.test):
+[Example file1](../../test/data/adult.tiny.with-schema.txt):
+[Example file2](../../test/data/adult.tiny.with-schema.txt):
```

Comment:
These are identical.

Reply:
We no longer have adult.train and adult.test in the repo. I'm using this file twice to demonstrate how multiple files can be loaded.

---
In reply to: [370990390](https://github.com/dotnet/machinelearning/pull/4706#discussion_r370990390) [](ancestors = 370990390)